client/src/components/Donation/donate-form.tsx

```diff
@@ -342,14 +349,16 @@ class DonateForm extends Component<DonateFormProps, DonateFormComponentState> {
             donationDuration={donationDuration}
             handlePaymentButtonLoad={this.handlePaymentButtonLoad}
             handleProcessing={handleProcessing}
-            isMinimalForm={isMinimalForm}
+            isMinimalForm={showMinimalPayments}
             isPaypalLoading={loading.paypal}
             isSignedIn={isSignedIn}
             onDonationStateChange={this.onDonationStateChange}
             theme={priorityTheme}
           />
-          <PatreonButton postPatreonRedirect={this.postPatreonRedirect} />
-          {isMinimalForm && (
+          {(!loading.stripe || !loading.paypal) && (
```

Comment:
We only show the patreon after one of the payment wallets are loaded to avoid the UI from jumping multiple times.